src/components/organisms/TimingEventModal/TimingEventModal.scss

```diff
@@ -13,6 +13,20 @@
     }
   }
 
+  &__duration_container {
+    input {
+      display: inline-block;
+      width: auto;
+      border: 0.1px solid var(--accent--over-20a);
```

Comment:
we have 
```scss
$border-width--none: 0px;
$border-width--slim: 1px;
$border-width--bold: 2px;
```

should this `0.1px` be a thing? 
If not, is it a workaround? 
If so, maybe a comment above it why it isn't simply `none`

Reply:
Existing styles. I'll update what's there to make it use the variables